pkg/apis/core/helper/helpers.go

```diff
@@ -164,7 +164,7 @@ func TaintsHave(taints []core.SeedTaint, key string) bool {
 }
 
 // TaintsAreTolerated returns true when all the given taints are tolerated by the given tolerations.
-func TaintsAreTolerated(taints []gardencorev1beta1.SeedTaint, tolerations []core.Toleration) bool {
+func TaintsAreTolerated(taints []core.SeedTaint, tolerations []core.Toleration) bool {
```

Comment:
Can you remove this and other functions in this file which are not used anymore?

Reply:
There are other functions as well which are unused in our code base we need to find them all and remove it. We should do the cleanup part in a separate PR.

Reply:
I meant just this file.